web/src/components/NewFeaturePopover/NewFeaturePopoverContent.tsx

```diff
@@ -0,0 +1,16 @@
+import { CalendarSearch } from 'lucide-react';
+import { useTranslation } from 'react-i18next';
+
+export function NewFeaturePopoverContent() {
+  const { t } = useTranslation();
+
+  return (
+    <div className="flex flex-col text-left">
+      <div className="flex flex-row items-center gap-2">
+        <CalendarSearch size={16} />
+        <h3>{t(`new-feature-popover.title`)}</h3>
+      </div>
+      <p>{t(`new-feature-popover.content`)}</p>
```

Comment:
Depending on how we decide to do the translation (see en.json comment), then it might be necessary to give the translation strings as props